src/gmenu2x.cpp

```diff
@@ -72,6 +73,10 @@
 #define DEFAULT_FONT_PATH "/usr/share/fonts/truetype/dejavu/DejaVuSansCondensed.ttf"
 #define DEFAULT_FONT_SIZE 12
 
+#ifndef DEFAULT_FALLBACK_FONTS
+#define DEFAULT_FALLBACK_FONTS ,{"/usr/share/fonts/truetype/droid/DroidSansFallbackFull.ttf",13},{"/usr/share/fonts/truetype/droid/DroidSansFallback.ttf",13}
+#endif
```

Comment:
I think it'd be better to disable fallback if DEFAULT_FALLBACK_FONTS is not set or empty.

Reply:
Done

Reply:
Reverted as discussed (for consistency with `DEFAULT_FONT_PATH`)